Sources/TokamakCore/Reflection/Models/PropertyInfo.swift

```diff
@@ -20,7 +20,20 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-public struct PropertyInfo {
+public struct PropertyInfo: Hashable {
+  public static func == (lhs: PropertyInfo, rhs: PropertyInfo) -> Bool {
+    lhs.name == rhs.name && lhs.type == rhs.type && lhs.isVar == rhs.isVar && lhs.offset == rhs
+      .offset && lhs.ownerType == rhs.ownerType
+  }
+
+  public func hash(into hasher: inout Hasher) {
+    hasher.combine(name)
+    hasher.combine(ObjectIdentifier(type))
+    hasher.combine(isVar)
+    hasher.combine(offset)
+    hasher.combine(ObjectIdentifier(ownerType))
+  }
```

Comment:
Aren't these implementations generated automatically by the compiler? If not, please add a comment above the functions to clarify that.